cmd/schema/main.go

```diff
@@ -173,15 +149,9 @@ func init() {
 	pflags.StringVar(&host, "host", host, "grip server url")
 	pflags.StringVar(&jsonFile, "json", "", "JSON graph file")
 	pflags.StringVar(&yamlFile, "yaml", "", "YAML graph file")
-
-	sflags := sampleCmd.Flags()
-	sflags.StringVar(&host, "host", host, "grip server url")
-	sflags.Uint32Var(&sampleCount, "sample", sampleCount, "Number of elements to sample")
-	sflags.BoolVar(&yaml, "yaml", yaml, "output schema in YAML rather than JSON format")
-	sflags.BoolVar(&manual, "manual", manual, "Use client side schema sampling")
-	sflags.StringSliceVar(&excludeLabels, "exclude-label", excludeLabels, "exclude vertex/edge label from schema")
+	pflags.StringVar(&jsonSchemaFile, "jsonSchema", "", "Json Schema")
+	pflags.StringVar(&graphName, "graphName", "", "Name of schemaGraph")
```

Comment:
For the other sub commands, graphName is defined via fixed argument and not a flag.